packages/teleterm/src/ui/TabHost/ClusterConnectPanel/ClusterConnectPanel.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { ButtonPrimary, Flex, Text } from 'design';
+import clusterPng from './clusters.png';
+import Image from 'design/Image';
+import { useAppContext } from 'teleterm/ui/appContextProvider';
+
+export function ClusterConnectPanel() {
+  const ctx = useAppContext();
+
+  function handleConnect() {
+    ctx.commandLauncher.executeCommand('cluster-connect', {});
```

Comment:
In `~/Library/Application\ Support/Electron/`, I temporarily renamed `app-state.json` and `tsh` to something else to have a clean app. After providing the cluster URI, I see this in the console:

> Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
>    in ClusterAdd (created by ClusterConnect)

Could you investigate that? Perhaps we don't need to handle it right now but it'd be worth at least having some faint idea on what causes this, maybe it's been here for months.

Anyway, after I provide my credentials, there's an error:

```
TypeError: Cannot read property 'findClusterDocument' of undefined
    at Object.run (app.js:26878)
    at CommandLauncher.executeCommand (app.js:26949)
    at Object.defaultHandler [as onSuccess] (app.js:26846)
    at onSuccess (app.js:14929)
    at app.js:15999
    at commitHookEffectListMount (vendor.js:56681)
    at commitPassiveHookEffects (vendor.js:56719)
    at HTMLUnknownElement.callCallback (vendor.js:37138)
    at Object.invokeGuardedCallbackDev (vendor.js:37187)
    at invokeGuardedCallback (vendor.js:37242)
```

This seems to be a bigger problem which we should fix before the demo.

Reply:
I pushed a fix for the second issue. The first one is more tricky, but not critical, I will add it to our master ticket.